### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by Hsintao** [microsoft/onnxruntime#2797]
**Describe the bug**
i build from source.
`./build.sh --update --build --cudnn_home /usr/local/cuda/lib64 --cuda_home /usr/local/cuda --use_tensorrt --tensorrt_home ~/xintao/TensorRT-6.0.1.5 --use_openvino CPU_FP32 --build_wheel`
**Urgency**
If there are particular important use cases blocked by this or strict project-related timelines, please share more information and dates. If there are no hard deadlines, please specify none.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):ubuntu 16.04
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: latest
- Python version: 3.7.5
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):7.4
- CUDA/cuDNN version:10.0 
- GPU model and memory: 2080ti 11G
- OPENVINO 2019R3.1
- TENSORRT 6.0

**To Reproduce**
Describe steps/code to reproduce the behavior:
import onnxruntime as ort
import numpy as np
import os
os.environ['CUDA_VISIBLE_DEVICES']= '1'
sess = ort.InferenceSession('./resnet50.onnx')

**Expected behavior**
run the code above,  it can print the optimization info.
2020-01-08 20:50:12.954057818 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.2.bn3.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954071716 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer1.2.bn3.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954088242 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer1.1.bn1.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954100556 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.0.bn3.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954113688 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.1.bn2.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954126135 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer3.0.bn2.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954143480 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer3.3.bn2.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954156415 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.2.bn2.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954168521 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.3.bn2.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954182316 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer1.1.bn3.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954195470 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.3.bn3.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954214170 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer3.0.bn3.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954229453 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer4.0.bn1.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954242735 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer3.1.bn1.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954255555 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.0.downsample.1.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954267995 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer3.5.bn2.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954280828 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer1.0.bn1.num_batches_tracked'. It is not used by any node and should be removed from the model.
2020-01-08 20:50:12.954296502 [W:onnxruntime:, graph.cc:2412 CleanUnusedInitializers] Removing initializer 'layer2.1.bn3.num_batches_tracked'. It is not used by any node and should be removed from the model.
Segmentation fault (core dumped)

**Comment by jywu-msft** [comment]
is this a TensorRT issue or OpenVINO or the combination of both?
Do you get this issue if you build without OpenVINO (TensorRT only)?

**Comment by Liujingxiu23** [comment]
@Hsintao Have you solve the problem? I compile onnxruntime only with openvino and have the similar problem with you. I use C++ API and model seem load successfully since the input dim printed is right. Buf core dumped when call "Run".

**Comment by jywu-msft** [comment]
> @Hsintao Have you solve the problem? I compile onnxruntime only with openvino and have the similar problem with you. I use C++ API and model seem load successfully since the input dim printed is right. Buf core dumped when call "Run".

can you please file a separate issue for OpenVINO with details on how to reproduce?
